src/Automation/Automation.Test/ScenarioTests/AutomationTests.cs

```diff
@@ -72,9 +72,11 @@ public void TestAutomationStartUnpublishedRunbook()
             TestRunner.RunTestScript("Test-AutomationStartUnpublishedRunbook -runbookPath ScenarioTests\\Resources\\Test-WorkFlowWithVariousParameters.ps1");
         }
 
-        [Fact(Skip = "Need x64 test framework.")]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Fact]
         [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
         public void TestAutomationRunbookWithParameter()
```

Comment:
This test fails in playback mode with below error. I need help from PowerShell team to fix the test.

Result StackTrace:	
at System.Management.Automation.Runspaces.PipelineBase.Invoke(IEnumerable input)
   at System.Management.Automation.PowerShell.Worker.ConstructPipelineAndDoWork(Runspace rs, Boolean performSyncInvoke)
   at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync)
   at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
   at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings)
   at System.Management.Automation.PowerShell.CoreInvoke[TOutput](IEnumerable input, PSDataCollection`1 output, PSInvocationSettings settings)
   at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings)
   at System.Management.Automation.PowerShell.Invoke()
   at Microsoft.WindowsAzure.Commands.ScenarioTest.EnvironmentSetupHelper.RunPowerShellTest(String[] scripts) in C:\PS\azure-powershell\tools\ScenarioTest.ResourceManager\EnvironmentSetupHelper.cs:line 572
   at Microsoft.Azure.Commands.TestFx.TestManager.RunTestScript(String[] scripts) in C:\PS\azure-powershell\tools\TestFx\TestManager.cs:line 219
   at Commands.Automation.Test.AutomationTests.TestAutomationRunbookWithParameter() in C:\PS\azure-powershell\src\Automation\Automation.Test\ScenarioTests\AutomationTests.cs:line 82
Result Message:	System.Management.Automation.ActionPreferenceStopException : The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Unable to find a matching HTTP request for URL 'PUT /subscriptions/d2b38167-d3ca-4d1f-a020-948eee21b6bc/resourceGroups/to-delete-01/providers/Microsoft.Automation/automationAccounts/fbs-aa-01/jobs/ae8b507a-e9da-458b-b175-1c258c855c7c?api-version=2017-05-15-preview'. Calling method Item()

Reply:
Skipped the test